### PR TITLE
KAW-7905: add new footer design v2

### PR DESCRIPTION
**Pull request by IoanaOrca**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix [#<gh-issue-id>](https://projects.netcentric.biz/browse/KAW-7905)

Test URLs:
- Before: https://main--edge-delivery-bimota--netcentric.hlx.page/drafts/ioana/test
- After: https://kaw-7905-bimota-v2-footer--edge-delivery-bimota--netcentric.hlx.page/drafts/ioana/test

### CODE REVIEW

**Review by lakshmishriaswathnarayana — CHANGES_REQUESTED** [review]
Screenshot 2024-09-05 at 12 25 50

1. Line height must be 150% can you add that ?
2. The social icons should be in row direction now it's column and gap between icons be 20px.

Screenshot 2024-09-05 at 12 32 07
Screenshot 2024-09-05 at 12 31 56

- The copyrights section has unnecessary margin at bottom, can you remove that ?

Screenshot 2024-09-05 at 12 38 04

Can you change it to 24px. the gap as per designs is 24px

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -1,125 +1,146 @@
 footer {
   background-color: #222;
```
Comment (lakshmishriaswathnarayana) [inline blocks/footer/footer.css]: Can you change this to `var(--black)`

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -1,125 +1,146 @@
...
-}
-
-.footer .footer-logo img {
-  width: 250px;
-  height: auto;
+.footer .columns-container .columns-wrapper {
+  padding-top: 40px;
 }
 
 .footer .footer-column > * {
```
Comment (lakshmishriaswathnarayana) [inline blocks/footer/footer.css:12]: Can you combine this and the next rule at 16

**Review by lakshmishriaswathnarayana — CHANGES_REQUESTED** [review]
The link in footer should have this state https://www.figma.com/design/USKRtMKMeHd7m1xpsG4aUE/Bimota?node-id=11-4333&m=dev 

The gap between the Listing heading and first list item is 24px and the gap between items in list should be 16px , please change that.

File: blocks/footer/footer.css (hunk 2 above)
Comment (IoanaOrca) [reply blocks/footer/footer.css:12]: great catch!

**Review by TomaszDziezykNetcentric — APPROVED** [review]

**Pull request merged**